src/sentry/ownership/grammar.py

```diff
@@ -112,6 +114,18 @@ def test_path(self, data):
 
         return False
 
+    def test_module(self, data):
+        for frame in _iter_frames(data):
+            filename = frame.get("module")
```

Comment:
rename this to `module`?

Could also potentially factor this to use the same comparison code as `test_path` and just pass in a list of dict keys to test on?

Reply:
Refactored, they both now use a `test_frames` function :-)